test/test_nn.py

```diff
@@ -363,7 +362,7 @@ def _do_test(self, test_case, module, input):
                 input = input.half().cuda()
                 module.half().cuda()
                 module(input)
-                for o in module.parameters():
+                for p in module.parameters():
```

Comment:
Huh, how did this ever work. Nice catch.

Reply:
lol, I thought this was intended and was going to change it back... previously this for loop uses the last p from the for loop above I guess :D